packages/amplify_api/lib/src/graphql/helpers/model_queries.dart

```diff
@@ -14,13 +14,14 @@
  */
 
 import 'package:amplify_api/amplify_api.dart';
-import 'package:amplify_api/graphql_helpers.dart';
+import 'package:amplify_api/src/graphql/factories/model_queries_factory.dart';
 import 'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_interface.dart';
 
-// This class provides static method calls to enable a simpler DX
+// This class provides static method calls to enable a simpler DX,
+// while preserving internal interfaces
```

Comment:
This file itself can be in lib (not in src) I think as it's a public facing API and included as export. The internal dependencies it imports, like model_queries_factory, are correct to be in src.

Reply:
Moved.